docs/src/main/sphinx/connector/hive.rst

```diff
@@ -572,6 +572,14 @@ Property Name                              Description
    * - ``hive.metastore.thrift.delete-files-on-drop``
      - Actively delete the files for drop table operations, for cases when the
        metastore does not delete the files. Default is ``false``.
+   * - ``hive.metastore.thrift.update-partition-statistics.batch.enabled``
+     - Update partition statistics using Hive thrift batch API. This can be
+       used to speed up the analysis of partitioned tables. Default is ``true``.
+   * - ``hive.metastore.thrift.update-partition-statistics.batch-size``
```

Comment:
We could have `hive.metastore.thrift.update-partition-statistics.batch-size` when set to `1` to correspond to no batch functionality. In this way we could get rid of `hive.metastore.thrift.update-partition-statistics.batch.enabled` and have in this fashion less configurations to deal with in Hive.